src/utils/jwt.js

```diff
@@ -39,15 +39,22 @@ export function isValidJwtToken(jwtToken: string): boolean {
   return JWT_TOKEN_RE.test(jwtToken);
 }
 
+const BASE64URL_REPLACEMENTS = {
+  '-': '+',
+  _: '/',
+};
+const BASE64URL_REPLACEMENT_RE = /[-_]/g;
```

Comment:
I didn't know that using `/g` global modifier on a regexp would replace all the occurrences in a text with `replace`. I assumed we would need `replaceAll`, but apparently it works with it too!